src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -38,7 +38,7 @@ mod dump_mir;
 mod error_reporting;
 mod graphviz;
 mod values;
-use self::values::{RegionValueElements, RegionValues};
+crate use self::values::{RegionElement, RegionElementIndex, RegionValueElements, RegionValues};
```

Comment:
In this code, I've been trying to stick to a style that avoids `crate use` and I'd prefer to keep doing so. In particular, I want to have things have **just one home** and that is where you import them from (unlike the rest of the compiler where I cannot tell, based on the filename where I find some code, what path I am supposed to use).

Reply:
Fixed this.